Libraries/LibWeb/EncryptedMediaExtensions/EncryptedMediaExtensions.idl

```diff
@@ -0,0 +1,6 @@
+// https://w3c.github.io/encrypted-media/#dom-mediakeysrequirement
+enum MediaKeysRequirement {
+    "required",
+    "optional",
+    "not-allowed"
+};
```

Comment:
missing newline at EOF. editor settings generally make this harder to forget

Reply:
Done. I was under the impression that the pre-commit checks would warn me about this.